.devcontainer/post-create.sh

```
@@ -0,0 +1,49 @@
+    echo "📋 Creating .env file from .env.example..."
+    cp .env.example .env
+fi
+
+# Install Python dependencies
+echo "📦 Installing Python dependencies..."
+pip install --upgrade pip
+pip install -r requirements.txt
+
+# Wait for database to be ready
+echo "⏳ Waiting for PostgreSQL to be ready..."
+until PGPASSWORD=$POSTGRES_PASSWORD psql -h db -U $POSTGRES_USER -d $POSTGRES_DB -c '\q' 2>/dev/null; do
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)